Classes/Toolbar/FLEXExplorerToolbar.m

```diff
@@ -147,28 +147,28 @@ - (void)layoutSubviews
     descriptionLabelFrame.size.height = kDescriptionLabelHeight;
     descriptionLabelFrame.origin.x = descriptionOriginX;
     descriptionLabelFrame.origin.y = kDescriptionVerticalPadding;
-    descriptionLabelFrame.size.width = CGRectGetMaxX(self.selectedViewDescriptionContainer.bounds) - kHorizontalPadding - descriptionOriginX;
-    self.selectedViewDescriptionLabel.frame = descriptionLabelFrame;
+    descriptionLabelFrame.size.width = CGRectGetMaxX(self.selectedItemDescriptionContainer.bounds) - kHorizontalPadding - descriptionOriginX;
+    self.selectedItemDescriptionLabel.frame = descriptionLabelFrame;
 }
 
 
 #pragma mark - Setter Overrides
 
-- (void)setSelectedViewOverlayColor:(UIColor *)selectedViewOverlayColor
+- (void)setselectedItemOverlayColor:(UIColor *)selectedItemOverlayColor
```

Comment:
Here's the source of the toolbar info not showing up. Should be `setSelectedItemOverlayColor` (capitalization)